src/main/kotlin/no/nav/syfo/persistering/db/PersisterDialogmeldingQueries.kt

```diff
@@ -37,8 +37,8 @@ fun ResultSet.toDialogmeldingTidspunkt(): DialogmeldingTidspunkt =
         mottattTidspunkt = getTimestamp("mottatt_tidspunkt").toLocalDateTime(),
     )
 
-private fun Connection.opprettDialogmeldingOpplysninger(receivedDialogmelding: ReceivedDialogmelding) {
-    this.prepareStatement(
+fun Connection.opprettDialogmeldingOpplysninger(receivedDialogmelding: ReceivedDialogmelding): String {
```

Comment:
Dette blir gjort for å bruke i en test. Kan også lage en identisk en som returnerer IDen i `TestDatabase`, og ikke røre denne, men følte endringen var grei nok og da kunne den leve her? 🤷🏼‍♂️

Reply:
Tenker det er en bra forbedring 👍